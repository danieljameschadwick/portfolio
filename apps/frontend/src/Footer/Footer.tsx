export const Footer = () => {
    return (
        <footer className={"footer-container bg-gray-dark"}>
            <div className={"site-info"}>
                <div className={"footer-brand"}>
                    <img src={"img/brand_logo.png"} width={30} height={30} alt={"Daniel Chadwick's logo."} />

                    <span>Daniel Chadwick</span>
                </div>

                <p>
                    Powered by NextJS, and MongoDB.

                    <br />

                    <small>Built with TailwindCSS.</small>
                </p>
            </div>
        </footer>
    );
};
