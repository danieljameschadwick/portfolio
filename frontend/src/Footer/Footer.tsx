export const Footer = () => {
    return (
        <footer className={"footer-container bg-gray-dark"}>
            <div className={"site-info"}>
                <div className={"footer-brand"}>
                    <img src={"img/brand_logo.png"} width={30} height={30} alt={"Daniel Chadwick logo."} />

                    <span>Daniel Chadwick</span>
                </div>

                <p>
                    Powered by NextJS, NestJS, PostgreSQL.

                    <br />

                    <small>Built with TailwindCSS.</small>
                </p>
            </div>
        </footer>
    );
};
