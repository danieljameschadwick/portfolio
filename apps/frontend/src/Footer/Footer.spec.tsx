import { render } from "@testing-library/react";
import { Footer } from "./Footer";

describe("Footer component", () => {
  it("renders base", () => {
    const { baseElement } = render(
      <Footer />
    );

    expect(baseElement).toMatchSnapshot();
  });
});
