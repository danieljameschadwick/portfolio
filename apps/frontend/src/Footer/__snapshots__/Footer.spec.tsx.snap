// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component renders base 1`] = `
<body>
  <div>
    <footer
      class="footer-container bg-gray-dark"
    >
      <div
        class="site-info"
      >
        <div
          class="footer-brand"
        >
          <img
            alt="Daniel Chadwick's logo."
            height="30"
            src="img/brand_logo.png"
            width="30"
          />
          <span>
            Daniel Chadwick
          </span>
        </div>
        <p>
          Powered by NextJS, and MongoDB.
          <br />
          <small>
            Built with TailwindCSS.
          </small>
        </p>
      </div>
    </footer>
  </div>
</body>
`;
